csm/conf/setup.py

```diff
@@ -832,6 +812,10 @@ def reset(self, args):
                 self._config_user_permission(reset=True)
                 self.Config.delete()
                 self._config_user(reset=True)
+                UDSConfigGenerator.delete()
+                cls = self.__class__
+                cls._restart_uds()
+                cls._restart_haproxy()
```

Comment:
@ajaysrivas @kupranay Please let me know if we need to restart services at the `reset` step. If we don't, I will remove these lines.